include/outputs

```diff
@@ -71,249 +38,226 @@ PROWLER_PARAMETERS=$@
 # $ACCOUNT_DETAILS_ORG
 # $ACCOUNT_DETAILS_TAGS
 
+EXTENSION_CSV="csv"
+EXTENSION_JSON="json"
+EXTENSION_ASFF="asff.json"
+EXTENSION_TEXT="txt"
+EXTENSION_HTML="html"
+HTML_LOGO_URL="https://github.com/prowler-cloud/prowler/"
+HTML_LOGO_IMG="https://github.com/prowler-cloud/prowler/raw/master/util/html/prowler-logo-new.png"
+TIMESTAMP=$(get_iso8601_timestamp)
+PROWLER_PARAMETERS=$@
 
 
-# Ensure that output directory always exists when -M is used
-if [[ $MODE ]];then
-  mkdir -p "${OUTPUT_DIR}"
-  if [[ "${MODES[@]}" =~ "html" ]]; then
-    addHtmlHeader > ${OUTPUT_FILE_NAME}.$EXTENSION_HTML
-    HTML_REPORT_INIT="1"
+# Set exitcode for codebuild/jenkins-like integrations
+set_exitcode () {
+  if [ "$FAILED_CHECK_FAILED_SCAN" == 1 ] && [ -z "$FAILED_CHECK_FAILED_SCAN_LIST" ]
+  then
+      EXITCODE=3
   fi
-  if [[ "${MODES[@]}" =~ "csv" ]]; then
-    printCsvHeader
+  if [[ "${FAILED_CHECK_FAILED_SCAN_LIST[@]}" =~ "$CHECK_NAME" ]]
+  then
+      EXITCODE=3
   fi
-fi
-
-# textInfo "HTML report will be saved: ${OUTPUT_FILE_NAME}.$EXTENSION_HTML"
-# textInfo "JSON ASFF report will be saved: ${OUTPUT_FILE_NAME}.$EXTENSION_ASFF"
-# textInfo "CSV report will be saved: ${OUTPUT_FILE_NAME}.$EXTENSION_CSV"
-# textInfo "JSON report will be saved: ${OUTPUT_FILE_NAME}.$EXTENSION_JSON"
-
-if [[ $PROFILE == "" ]];then
-  PROFILE="ENV"
-fi
+}
 
-textPass(){
-  CHECK_RESULT="PASS"
-  CHECK_RESULT_EXTENDED="$1"
-  CHECK_RESOURCE_ID="$3"
+# Add header to certain output files readed from mode
+output_files_init() {
+  for MODE in "${MODES[@]}"; 
+  do
+    if  [ "${MODE}" == 'html' ] 
+    then
+        addHtmlHeader > "${OUTPUT_FILE_NAME}"."${EXTENSION_HTML}"
+        HTML_REPORT_INIT="1"
+    fi
+    if [ "${MODE}" == 'csv' ]
+    then
+        printCsvHeader
+    fi
+  done
+}
 
-  if [[ "$QUIET" == 1 ]]; then
-    return
-  fi
+# Check if resource checked is allowlisted
+allowlist_check() {
+  # Checked value is the whole log message that comes as argument
+  CHECKED_VALUE=${1}
+  # Default value of result is Fail
+  CHECK_RESULT='FAIL'
+  ## ignore allowlists for current check
+  while read -r excluded_item; do
+    # ignore_check_name is the check with resources allowlisted
+    IGNORE_CHECK_NAME=$(awk -F ":" '{print $1}' <<< "${excluded_item}")
+    # Resource value is what it comes after CHECK_NAME: :
+    RESOURCE_VALUE=$(awk -F "$CHECK_NAME:" '{print $2}' <<< "${excluded_item}")
```

Comment:
Solved here https://github.com/prowler-cloud/prowler/pull/1180/commits/bed5466a2b0ff87e120b725d79dca9ef3e834cc2

Reply:
Real solving here https://github.com/prowler-cloud/prowler/pull/1180/commits/d1a96640213db773920fdc0015a32bbfe3d2d19b IFS here not works because we need to split by `"$CHECK_NAME:"` in `RESOURCE_VALUE `